9-regular-expressions/09-regexp-quantifiers/article.md

```diff
@@ -1,46 +1,46 @@
 
-Let's say we have a string like `+7(903)-123-45-67` and want to find all numbers in it. But unlike before, we are interested not in single digits, but full numbers: `7, 903, 123, 45, 67`.
+Digamos que temos a string `+7(903)-123-45-67` em mãos e gostaríamos de encontrar todos os números dela. Entretanto, diferente de casos anteriores, não estamos interessados em dígitos soltos, mas sim nos números inteiros: `7, 903, 123, 45, 67`.
 
-A number is a sequence of 1 or more digits `pattern:\d`. To mark how many we need, we can append a *quantifier*.
+Um número é uma sequência de 1 ou mais dígitos `pattern:\d`. Para determinar quantos desses precisamos, usamos um *quantificador*.
 
-## Quantity {n}
+## Quantidade {n}
 
-The simplest quantifier is a number in curly braces: `pattern:{n}`.
+O quantificador mais simples é um número entre chaves: `pattern:{n}`.
 
-A quantifier is appended to a character (or a character class, or a `[...]` set etc) and specifies how many we need.
+Quantificadores são colocados após um caractere (ou uma classe de caracteres, ou um conjunto `[...]`, etc.) e especifica quantos do elemento anterior nós precisamos.
 
-It has a few advanced forms, let's see examples:
+Ele possui algumas formas avançadas, vejamos alguns exemplos:
 
-The exact count: `pattern:{5}`
-: `pattern:\d{5}` denotes exactly 5 digits, the same as `pattern:\d\d\d\d\d`.
+Quantidade exata: `pattern:{5}`
+: `pattern:\d{5}` representa exatamente 5 dígitos, idêntico a `pattern:\d\d\d\d\d`.
 
-    The example below looks for a 5-digit number:
+    O exemplo abaixo procura por um número de 5 dígitos:
 
     ```js run
-    alert( "I'm 12345 years old".match(/\d{5}/) ); //  "12345"
+    alert( "Eu tenho 12345 anos de idade".match(/\d{5}/) ); //  "12345"
     ```
 
-    We can add `\b` to exclude longer numbers: `pattern:\b\d{5}\b`.
+    Podemos usar o `\b` para não reconhecer números maiores: `pattern:\b\d{5}\b`.
 
-The range: `pattern:{3,5}`, match 3-5 times
-: To find numbers from 3 to 5 digits we can put the limits into curly braces: `pattern:\d{3,5}`
+Um alcance: `pattern:{3,5}`, repetição de 3 a 5 vezes
+: Para encontrar números de 3 a 5 dígitos, podemos usar os limites entre chaves e separados por uma vírgula: `pattern:\d{3,5}`
 
     ```js run
-    alert( "I'm not 12, but 1234 years old".match(/\d{3,5}/) ); // "1234"
+    alert( "Não tenho 12, mas sim 1234 anos de idade".match(/\d{3,5}/) ); // "1234"
     ```
 
-    We can omit the upper limit.
+    Também podemos omitir o limite máximo.
 
-    Then a regexp `pattern:\d{3,}` looks for sequences of digits of length `3` or more:
+    Dessa forma, a expressão `pattern:\d{3,}` reconhece qualquer número com no mínimo 3 dígitos:
 
     ```js run
-    alert( "I'm not 12, but 345678 years old".match(/\d{3,}/) ); // "345678"
+    alert( "Não tenho 12, mas sim 345678 anos de idade".match(/\d{3,}/) ); // "345678"
     ```
 
-Let's return to the string `+7(903)-123-45-67`.
+Vamos voltar à string `+7(903)-123-45-67`.
```

Comment:
To make it shorter, we can translate it as `Voltaremos à string`. But the original is good enough, so I'm leaving my comments as a suggestion.